style.css

```diff
@@ -267,3 +267,170 @@ h5 {
     }
   }
 }
+
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+  width: 100%;
+  top: 72rem;
+  height: 44rem;
+  background-image: url(image/contact\ form\ shapes-mobile.png);
+  background-color: #396df2;
+  background-repeat: no-repeat;
+  background-size: cover;
+  overflow-x: hidden !important;
+  border-top-left-radius: 10%;
+  margin: 0 auto;
+  padding: 1em;
+  border: 1px solid #ccc;
+}
+
+.form ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.form ul li {
+  margin-top: 1em;
+}
+
+.contact-sec h2 {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-size: 40px;
+  font-weight: 700;
+  font-stretch: wider;
+  line-height: 52px;
+  position: absolute;
+  width: 65%;
+  height: 56px;
+  color: #fff;
+  top: 123px;
+  left: 66px;
+}
+
+.contact-sec p {
+  text-align: center;
+  font-family: "Poppins", sans-serif;
+  font-size: 20px;
+  font-weight: 400;
+  font-style: normal;
+  line-height: 28px;
+  font-stretch: wider;
+  position: absolute;
+  height: 10rem;
+  width: 90%;
+  top: 188px;
+  left: -12px;
+  color: #ebebff;
+}
+
+.name,
+.email,
+.msg {
+  position: absolute;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  background: #fff;
+}
+
+#name {
+  display: flex;
+  align-items: center;
+  font-family: 'Poppins', sans-serif;
+  position: absolute;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  padding: 10px;
```

Comment:
- [ ] Kindly make sure to follow the design guideline.
- [ ] Remove the padding for #name #email #msg so that you don't get the form overlay
- Your Design
<img width="316" alt="Screenshot 2023-06-09 at 05 45 48" src="https://github.com/BilalLiaquat7/Portfolio/assets/81444338/4371ac92-d9d9-4959-a36a-fb81c9241126">